### ISSUE TITLE
Focus indicator not visible/uniform

### ISSUE THREAD

**Issue by fabianhueske** [shopware/meteor#815]
### Package

@meteor/component-library

### Package version

Latest

### Steps to reproduce

Use keyboard navigation to focus interactive UI elements.

### Actual behavior

Only some elements like checkboxes and switches use a proper focus outline that's visible and visually pleasing.

### Expected behavior

All interactive elements use the same focus outline with offset and blue color.
